wpimath/src/main/native/include/WPIMathExports.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#ifdef _WIN32
+#ifdef WPILIB_BUILDING
+#define WPIMATH_DLLEXPORT __declspec(dllexport)
+#else
+#define WPIMATH_DLLEXPORT
+#endif
+#else
+#define WPIMATH_DLLEXPORT
+#endif

```

Comment:
```suggestion
#if defined(_WIN32) && defined(WPILIB_BUILDING)
#define WPIMATH_DLLEXPORT __declspec(dllexport)
#else
#define WPIMATH_DLLEXPORT
#endif

```

Reply:
We could go further and add Linux support here.  https://gcc.gnu.org/wiki/Visibility (we could remove the GCC version check).